CMakeLists.txt

```diff
@@ -114,6 +114,12 @@ function(spvtools_default_compile_options TARGET)
     endif()
   endif()
 
+  if (MSVC)
+    # Specify /EHsc for exception handling. This makes using SPIRV-Tools as
+    # dependencies in other projects easier.
+    target_compile_options(${TARGET} PRIVATE /EHsc)
```

Comment:
So what happens if an allocation failure occurs deep inside SPIR-V Tools. Then an exception is thrown.  But most of the SPIRV-Tools API is extern "C".  What happens?